web/src/components/tableOfContents.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import * as utils from '../utils'
+
+const WithAnchor = ({ children, slug }) => (
+  <Link to={`#${slug}`}>{children}</Link>
+)
+
+const TableOfContents = ({ blocks }) => {
+  const targetItems = ['h1', 'h2', 'h3', 'h4']
+  const menuItems = blocks.filter((block) => targetItems.includes(block.style))
+
+  if (menuItems.lenght === 0) {
+    return null
+  }
```

Comment:
This is misspelled and therefore always false :)
```suggestion
  if (!menuItems.length) {
    return null
  }
```